src/locales/fr/LC_MESSAGES/messages.po

```diff
@@ -2,60 +2,361 @@
+msgid "🚶🏽‍♀️ Main Menu 🗄️"
+msgstr "🚶🏽‍♀️ Menu principal 🗄️"
+
+#: services/messenger.py:366
+msgid "Discover GitHub"
+msgstr "Découvre GitHub"
+
+#: services/messenger.py:368
+msgid "Discover GitHub official website, or follow a beginner tutorial"
+msgstr "Découvre le site officiel de GitHub, ou suis un tutoriel pour débutant"
+
+#: services/messenger.py:381
+msgid "Good question 👌🏽"
+msgstr "Bonne question 👌🏽"
+
+#: services/messenger.py:385
+msgid ""
+"Git is a type of version control system (VCS) that makes it easier to "
+"track changes to files. "
+msgstr ""
+"Git est un type de système de contrôle de version (VCS) qui facilite le "
+"suivi des modifications apportées aux fichiers. "
+
+#: services/messenger.py:393
+msgid ""
+"For example, when you edit a file, Git can help you determine exactly "
+"what changed, who changed it, and why."
+msgstr ""
+
+#: services/messenger.py:402
+msgid "👶🏽 Install Git"
+msgstr "👶🏽 Installer Git"
+
+#: services/messenger.py:406
+msgid "🤓 I've Git Installed"
+msgstr "🤓 J'ai installé Git"
+
+#: services/messenger.py:411
+msgid "Want to learn more about Git?"
+msgstr "Tu veux en savoir plus sur Git ?"
+
+#: services/messenger.py:419
+msgid "Time to get git installed in your machine ⭕!."
+msgstr "Il est temps d'installer Git sur ta machine ⭕ !."
+
+#: services/messenger.py:425
+msgid "Download Git"
+msgstr "Télécharger Git"
+
+#: services/messenger.py:429
+msgid "Head over here, and donload git client based on your OS."
+msgstr ""
+"Rendez-vous ici et télécharge le client Git en fonction de ton système "
+"d'exploitation."
+
+#: services/messenger.py:440
+msgid "Configure Git ⚒️"
+msgstr "Configurer Git ⚒️"
+
+#: services/messenger.py:445
+msgid "🧑‍🚀 Once done, let's configure Git"
+msgstr ""
+
+#: services/messenger.py:453
+msgid "Great Progress so far 👨🏽‍🎓!."
+msgstr "De grands progrès jusqu'à présent 👨🏽‍🎓 !."
+
+#: services/messenger.py:457
+msgid ""
+"Now let's configure your Git username and email using the following "
+"commands"
+msgstr ""
+"Maintenant, configure ton nom d'utilisateur et ton e-mail Git à l'aide "
+"des commandes suivantes"
+
+#: services/messenger.py:463
+msgid "`$ git config --global user.name \"Steve Josh\"`"
+msgstr ""
+
+#: services/messenger.py:468
+msgid "`$ git config --global user.email \"josh@example.com\"`"
+msgstr ""
+
+#: services/messenger.py:473
+msgid "don't forget to replace Steve's name with your own."
+msgstr "n'oublie pas de remplacer le nom Steve par le tien"
+
+#: services/messenger.py:478
+msgid "These details will be associated with any commits that you create"
+msgstr "Ces détails seront associés à tous les commits que tu feras"
+
+#: services/messenger.py:491
+msgid "Now let's check what is Github👇🏼"
+msgstr "Voyons maintenant ce qu'est Github👇🏼"
+
+#: services/messenger.py:500
+msgid "Facebook 🧡 Open Source!"
+msgstr "Facebook 🧡 l'Open Source !"
+
+#: services/messenger.py:504
+msgid ""
+"Facebook maintains a lot of Open Source projects ranging from:\n"
```

Comment:
I think we also need to tackle issues #14 in this pull request to maintain the consistent content with [Facebook Open Source Projects website](https://opensource.facebook.com/projects)